tests/extensions/test_files_store.py

```diff
@@ -70,6 +73,30 @@ def test_spider_closed_even(caplog):
         ]
 
 
+def test_spider_closed_no_data(caplog):
+    spider = spider_with_files_store('no_data')
```

Comment:
Use tmpdir since test should still pass even if FILES_STORE exists.

```suggestion
def test_spider_closed_no_data(tmpdir, caplog):
    spider = spider_with_files_store(tmpdir)
```